applications: nrf_desktop: Change ts resolution to us in motion sim (#11853)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    This is done to prevent the device from powering down and waking up multiple times when an USB cable is connected.
  * Disabled ``CONFIG_BOOT_SERIAL_IMG_GRP_HASH`` in MCUboot bootloader release configurations of boards that use nRF52820 SoC.
    This is done to reduce the memory consumption.

>   * Motion generation in the Motion Simulated module is now based on the cycle count and uses a timestamp in microseconds instead of a timestamp in milliseconds to improve accuracy.

Reviewer: Add ref to module's doc page, please. also consider linking to `k_cycle_get_32` and `k_cycle_get_64` APIs to make it clear.

>   * Simulated movement data generation in the :ref:`nrf_desktop_motion` is now based on the cycle count (:c:func:`k_cycle_get_32` or :c:func:`k_cycle_get_64` function depending on the :kconfig:option:`CONFIG_TIMER_HAS_64BIT_CYCLE_COUNTER` Kconfig option) and uses a timestamp in microseconds instead of a timestamp in milliseconds to improve accuracy.

Reviewer 2: ```suggestion
  * Generating simulated movement data in the :ref:`nrf_desktop_motion` is now based on the cycle count (either :c:func:`k_cycle_get_32` or :c:func:`k_cycle_get_64` function depending on the :kconfig:option:`CONFIG_TIMER_HAS_64BIT_CYCLE_COUNTER` Kconfig option).
  Additionally, timestamp is outputted in microseconds instead of milliseconds to improve the accuracy.
```
Rephrased a bit and split the sentence to enhance the flow, please let me know if this affects the meaning in any way.

Author: Information about changing timestamp to microseconds and using cycle count is connected with each other (as timestamp in us depends on the usage of cycle count) so it changes meaning a bit.

Now I think that we could drop the information of previously used "timestamp in milliseconds" to simplify it and maybe rearrange the original sentence to be more precise about the change in implementation that happened.
Something like:
```suggestion
  * The generation of simulated movement data in the :ref:`nrf_desktop_motion` now uses a timestamp in microseconds based on the cycle count (either :c:func:`k_cycle_get_32` or :c:func:`k_cycle_get_64` function depending on the :kconfig:option:`CONFIG_TIMER_HAS_64BIT_CYCLE_COUNTER` Kconfig option) to improve the accuracy.
```

What do you think?

Reviewer 2: I think it's better :) though I still feel too much is happening at the end of the sentence. What if we moved the last part of the sentence to the beginning? In other words:
To improve the accuracy, the generation of simulated movement data in the :ref:`nrf_desktop_motion` now uses a timestamp in microseconds based on the cycle count (either :c:func:`k_cycle_get_32` or :c:func:`k_cycle_get_64` function depending on the :kconfig:option:`CONFIG_TIMER_HAS_64BIT_CYCLE_COUNTER` Kconfig option).  (?)

Author: Sure, even better!
